### ISSUE TITLE
gomobile tool

### ISSUE THREAD

**Issue by 0x4139** [anacrolix/torrent#152]
Trying to compile the master branch of this project using gomobile tool, i know there is a mobile branch but this has more commits ahead and i am getting the following error
```
00096 ([my path here]/github.com/anacrolix/torrent/client.go:711)      MOVD    "".extensions+44(FP), R1

```

i have no clue to what is this refering to

**Comment by axet** [comment]
fixed in go 1.8.1

**Comment by 0x4139** [comment]
Thank you very much!. updating go solved my issue
